sale_timesheet_task_exclude/tests/test_sale_timesheet_exclude_task.py

```diff
@@ -116,16 +116,13 @@ def test_exclude_from_sale_order(self):
                 "employee_id": employee.id,
             }
         )
-        self.assertTrue(timesheet.so_line)
-        self.assertEqual(task.billable_type, "task_rate")
+        self.assertTrue(task.sale_order_id)
 
         task.exclude_from_sale_order = True
-        self.assertEqual(task.billable_type, "no")
-        self.assertFalse(timesheet.so_line)
+        self.assertFalse(task.sale_order_id)
 
         task.exclude_from_sale_order = False
-        self.assertEqual(task.billable_type, "task_rate")
-        self.assertTrue(timesheet.so_line)
+        self.assertTrue(task.sale_order_id)
```

Comment:
I don't really follow this change, you don't check that timesheets generated from the task became non-billable.

Reply:
In v14 it is has changes so I need to do it.

Reply:
But the change is dramatic, if timesheet still has `so_line` on it. Please explain

Reply:
Yes if the task has allow_billable  than timesheet will have no so_line. So I have updated the testcase. @alexey-pelykh can you please check now.